src/test/java/bio/terra/janitor/app/controller/UnauthenticatedApiControllerTest.java

```diff
@@ -25,13 +30,13 @@
 
   @Test
   public void testStatusOK() throws Exception {
-    assertEquals(
-        "{\"ok\":true,\"systems\":{\"postgres\":{\"ok\":true}}}",
-        this.mvc
-            .perform(get("/status"))
-            .andExpect(status().isOk())
-            .andReturn()
-            .getResponse()
-            .getContentAsString());
+    MockHttpServletResponse response =
+        this.mvc.perform(get("/status")).andExpect(status().isOk()).andReturn().getResponse();
+
+    SystemStatus status =
+        new ObjectMapper().readValue(response.getContentAsString(), SystemStatus.class);
+    assertTrue(status.isOk());
+    assertThat(status.getSystems(), Matchers.hasKey("postgres"));
+    assertThat(status.getSystems(), Matchers.hasKey("stairway"));
```

Comment:
Would it be easier to call `/shutDown` then` /status` then we can verify both?

Reply:
Not sure what you mean about ordering. Did add a test that helped find a bug in my application.properties, thanks.

Reply:
I mean since stairway already have a nice easy to shutdown things, then we can test the shutDown endpoint along with the status. i.e., add a test:
``` @Test
  public void tesShutDownAndServiceUnHealthy() throws Exception {       
     this.mvc.perform(get("/shutDown")).andExpect(status().isOk()).andReturn().getResponse();
    // Then get status endpoint and verify stairWay is not health..   
        this.mvc	
            .perform(get("/status"))	   
        SystemStatus status =
            .andExpect(status().**isNotOk**());```

Reply:
done.